C# Examples/EposServiceConsumer/README.md

```diff
@@ -0,0 +1,92 @@
+# EPOSServiceConsumer
+
+This solution contains a series of Unit tests that should help you better understand the EPOS API implementation.
+You will require the following information to pass through to environment variables that can be edited in the [App.config](https://github.com/ResDiary/ResDiary-OAuth-Examples/blob/RD-17292/C%23%20Examples/EposServiceConsumer/UnitTests/App.config) file that belongs to the Unit Tests of the solution.
+
+| Variable Names | Description |
+| --- | --- |
+| `EPOS_CONSUMER_KEY` | A key that is a standard part of oAuth authorization|
+| `EPOS_CONSUMER_SECRET`| The first secret for the restaurant you are trying to access.Varies per restaurant  |
+|`EPOS_SERVICE_OAUTH_URI` | https://app.rdbranch.com/OAuth/V10a |
+|`EPOS_SERVICE_BASE_URI` |http://app.rdbranch.com |
+|`EPOS_API_SANDBOX_SECOND_SECRET`|The second secret for the restaurant you are trying to access.Varies per restaurant|
```

Comment:
Can you rename these sandbox ones to `EPOS_API_RESTAURANT_ID`, etc?

The reason we had the `API_SANDBOX` and `API_SANDBOX_WITH_BUMPING` was that we had multiple restaurants setup with different configurations for running integration tests against the API. That's specific to our integration tests, and 3rd parties don't need to care about that.

I'd just keep one set of the credentials / Ids, and you can probably get rid of the segment Id, area Id, table Ids, and promotion Ids. Once again this was for our integration tests, and not really relevant for someone just trying to get up and running with the API.